pkg/securitypolicy/securitypolicyenforcer.go

```diff
@@ -233,12 +246,68 @@ func (policyState *StandardSecurityPolicyEnforcer) EnforceCommandPolicy(containe
 		return errors.New(errmsg)
 	}
 
-	// record that we've allowed this container to start
-	policyState.startedContainers[containerID] = struct{}{}
+	return nil
+}
+
+func (policyState *StandardSecurityPolicyEnforcer) enforceEnvironmentVariablePolicy(containerID string, envList []string) (err error) {
+	// Get a list of all the indexes into our security policy's list of
+	// containers that are possible matches for this containerID based
+	// on the image overlay layout and command line
+	possibleIndexes := possibleIndexesForID(containerID, policyState.ContainerIndexToContainerIds)
+
+	for _, envVariable := range envList {
+		matchingRuleFoundForSomeContainer := false
+		for _, possibleIndex := range possibleIndexes {
+			envRules := policyState.SecurityPolicy.Containers[possibleIndex].EnvRules
+			ok := envIsMatchedByRule(envVariable, envRules)
+			if ok {
+				matchingRuleFoundForSomeContainer = true
+			} else {
+				// a possible matching index turned out not to match, so we
+				// need to update that list and remove it
+				policyState.narrowMatchesForContainerIndex(possibleIndex, containerID)
+			}
+		}
+
+		if !matchingRuleFoundForSomeContainer {
+			errmsg := fmt.Sprintf("env variable %s unmatched by policy rule", envVariable)
+			return errors.New(errmsg)
```

Comment:
can just `return fmt.Errorf(<same thing in the fmt.Sprintf>)`

Reply:
done